app/components/works/subtypes_component.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="mb-3 row">
   <div class="col-sm-12 h5">
-    Work types <%= '(optional)' if !other_type? %>
+    Work types <%= '(optional)' if optional? %>
```

Comment:
We'll also need this when we fix mixed materials to require two or more primary subtypes.